pkgs/development/python-modules/torchvision/default.nix

```diff
@@ -1,30 +1,41 @@
-{ lib
-, symlinkJoin
-, buildPythonPackage
+{ buildPythonPackage
+, cudaSupport ? torch.cudaSupport or false # by default uses the value from torch
 , fetchFromGitHub
-, ninja
-, which
+, lib
 , libjpeg_turbo
 , libpng
+, ninja
 , numpy
-, scipy
 , pillow
-, torch
 , pytest
-, cudaSupport ? torch.cudaSupport or false # by default uses the value from torch
+, scipy
+, symlinkJoin
+, torch
+, which
 }:
 
 let
-  inherit (torch) gpuTargetString;
-  inherit (torch.cudaPackages) cudatoolkit cudnn;
+  inherit (torch) cudaPackages gpuTargetString;
+  inherit (cudaPackages) cudatoolkit cudaFlags cudaVersion;
 
-  cudatoolkit_joined = symlinkJoin {
-    name = "${cudatoolkit.name}-unsplit";
-    paths = [ cudatoolkit.out cudatoolkit.lib ];
+  cuda-redist = symlinkJoin {
+    name = "cuda-redist-${cudaVersion}";
+    # NOTE: torchvision doesn't actually use cudnn; torch does!
+    paths = with cudaPackages; [
+      cuda_nvcc
+      cuda_cudart # cuda_runtime.h
+      cuda_cccl # <thrust/*>
+      libcublas
+      libcusparse
+      libcusolver
+    ];
   };
-in buildPythonPackage rec {
-  pname = "torchvision";
+
+  name = "torchvision";
   version = "0.14.1";
+in
+buildPythonPackage {
+  inherit name version;
```

Comment:
please use `pname` instead of `name`. `name` has been deprecated: https://github.com/NixOS/nixpkgs/issues/103997

Reply:
Oh wow! I had wondered why I saw the two of them, but hadn't thought to ask. Will change!